frontend/kubecloud/src/utils/notificationUtils.ts

```diff
@@ -0,0 +1,77 @@
+import { formatDistanceToNow } from 'date-fns'
+import type { NotificationType, NotificationSeverity } from '../types/notifications'
+
+/**
+ * Get notification icon for different types
+ * @param type - Notification type
+ * @returns Icon name
+ */
+export function getNotificationIcon(type: NotificationType): string {
+  switch (type) {
+    case 'deployment': return 'mdi-rocket-launch'
+    case 'billing': return 'mdi-credit-card'
+    case 'user': return 'mdi-account'
+    case 'connected': return 'mdi-link'
+    case 'node': return 'mdi-server'
+    default: return 'mdi-bell'
+  }
+}
+
+/**
+ * Get notification color for different types
+ * @param type - Notification type
+ * @returns Color name
+ */
+export function getNotificationColor(type: NotificationType): string {
+  switch (type) {
+    case 'deployment': return 'success'
+    case 'billing': return 'warning'
+    case 'user': return 'info'
+    case 'connected': return 'primary'
+    case 'node': return 'info'
+    default: return 'grey'
+  }
+}
```

Comment:
i think the color should be related to the severity 

<img width="1268" height="697" alt="Image" src="https://github.com/user-attachments/assets/fdbd51aa-4f8f-4513-b974-a2f496b49f1b" />

in this i don't understood why ssh key is info and the failed deployment is success